shell-mommy.sh

```diff
@@ -84,22 +84,34 @@ mommy() {
   if [[ -n "${SHELL_MOMMYS_NEGATIVE_RESPONSES:-}" ]]; then
     NEGATIVE_RESPONSES=("${SHELL_MOMMYS_NEGATIVE_RESPONSES[@]}")
   fi
-  
+
+  # generate a random base 10 number using /dev/urandom
+  random() {
+    rd_number=$(tr -cd '[:digit:]' < /dev/urandom | head -c 6)
```

Comment:
```suggestion
    rd_number=$(LC_CTYPE=C tr -cd '[:digit:]' < /dev/urandom | head -c 6)
```
This allows macOS' zsh to work otherwise it throws a bunch of illegal byte sequence errors. I tested this to make sure it still worked on a Linux distro, I used Ubuntu, and it still works with that added.

Reply:
Oh wow, interesting, didn't know that. I'll add it to the MR